src/GitTools.Core/GitTools.Core.csproj

```diff
@@ -19,36 +19,36 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\output\debug\NET45\GitTools.Core\</OutputPath>
+    <OutputPath>..\..\output\Debug\GitTools.Core\net45\</OutputPath>
```

Comment:
Yeah, debug\proj\alltargets ?

Sent from my Windows Phone
________________________________
From: Geert van Horrik<mailto:notifications@github.com>
Sent: ‎11/‎09/‎2015 7:13 PM
To: GitTools/GitTools.Core<mailto:GitTools.Core@noreply.github.com>
Cc: Jake Ginnivan<mailto:jake@ginnivan.net>
Subject: Re: [GitTools.Core] Solution update (#15)


In src/GitTools.Core/GitTools.Core.csproj<https://github.com/GitTools/GitTools.Core/pull/15#discussion_r39261702>:

> @@ -19,36 +19,36 @@
>      <DebugSymbols>true</DebugSymbols>
>      <DebugType>full</DebugType>
>      <Optimize>false</Optimize>
> -    <OutputPath>..\..\output\debug\NET45\GitTools.Core\</OutputPath>
> +    <OutputPath>..\..\output\Debug\GitTools.Core\net45\</OutputPath>


GitTools.Core
GitTools.IssueTrackers
GitTools.Testing

I think we have more than 1 lib :-)

—
Reply to this email directly or view it on GitHub<https://github.com/GitTools/GitTools.Core/pull/15/files#r39261702>.